client/src/components/alerts/component.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { CheckCircleIcon, XCircleIcon, ExclamationIcon } from '@heroicons/react/solid';
+
+import { AlertsProps } from './types';
+
+const ALERT_CLASSES = {
+  success: {
+    iconClassnames: 'mr-2 h-5 w-5 fill-green-600',
+    backgroundColor: 'bg-green-100',
+    titleColor: 'text-green-800',
+    messagesColor: 'text-green-900',
+  },
+  error: {
+    iconClassnames: 'mr-2 h-5 w-5 fill-red-700',
+    backgroundColor: 'bg-red-100',
+    titleColor: 'text-red-800',
+    messagesColor: 'text-red-900',
+  },
+  warning: {
+    iconClassnames: 'mr-2 h-5 w-5 fill-yellow-500',
+    backgroundColor: 'bg-yellow-100',
+    titleColor: 'text-yellow-600',
+    messagesColor: 'text-yellow-900',
+  },
+};
+
+const Alerts: React.FC<AlertsProps> = ({ className, items }: AlertsProps) => {
+  if (!items) return null;
+  if (Array.isArray(items) && !items.length) return null;
+
+  const alerts = Array.isArray(items) ? items : [items];
+
+  const getIcon = (type) => {
```

Comment:
There's a way better way of doing this, but for the life of me I can't remember. Will get back to it later.